web/cobrands/northamptonshire/assets.js

```diff
@@ -618,4 +619,33 @@ function check_emergency() {
 }
 $(fixmystreet).on('report_new:category_change', check_emergency);
 
+function ncc_check_streetlights() {
+    var emergency_state = ncc_is_emergency_category();
+
+    if (emergency_state.relevant || fixmystreet.body_overrides.get_only_send() == 'Highways England') {
+        $('#ncc_streetlights').remove();
+        return;
+    }
+
+    var relevant_body = OpenLayers.Util.indexOf(fixmystreet.bodies, northants_defaults.body) > -1;
+    var relevant_cat = $('#form_category').val() == 'Street lighting';
+    var relevant = relevant_body && relevant_cat;
+    var currently_shown = !!$('#ncc_streetlights').length;
+
+  console.log(relevant, currently_shown);
+    if (!relevant) {
+        $('#ncc_streetlights').remove();
+        enable_report_form();
+        return;
+    }
+
+    if (!currently_shown) {
+        var $msg = $('<p id="ncc_streetlights" class="box-warning">Street lighting in Northamptonshire is maintained by Balfour Beatty on behalf of the County Council under a Street Lighting Private Finance Initiative (PFI) contract. Please view our <b><a href="https://www3.northamptonshire.gov.uk/councilservices/northamptonshire-highways/roads-and-streets/Pages/street-lighting.aspx">Street Lighting</a></b> page to report any issues.</p>');
+        $msg.insertBefore('#js-post-category-messages');
+        disable_report_form();
+        hide_responsibility_errors();
```

Comment:
Is this line necessary? Isn't it already handled by the removal of the asset/road layer from the category change?